mysql-ballerina/build.gradle

```diff
@@ -154,135 +158,127 @@ task initializeVariables {
-            from file("$project.projectDir/target/cache")
-            exclude '**/*-testable.jar'
-            exclude '**/tests_cache/'
-            into file("$artifactCacheParent/cache/")
-        }
 
-        // Doc creation and packing
-        exec {
-            workingDir project.projectDir
-            environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
-            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-                commandLine 'cmd', '/c', "$distributionBinPath/bal.bat doc && exit %%ERRORLEVEL%%"
-            } else {
-                commandLine 'sh', '-c', "$distributionBinPath/bal doc"
+            // Doc creation and packing
+            exec {
+                workingDir project.projectDir
+                environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
+                if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                    commandLine 'cmd', '/c', "$distributionBinPath/bal.bat doc && exit %%ERRORLEVEL%%"
+                } else {
+                    commandLine 'sh', '-c', "$distributionBinPath/bal doc"
+                }
             }
-        }
-        copy {
-            from file("$project.projectDir/target/apidocs/${packageName}")
-            into file("$project.projectDir/build/docs_parent/docs/${packageName}")
-        }
-    }
-
-    outputs.dir artifactCacheParent
-    outputs.dir artifactBallerinaDocs
-    outputs.dir artifactLibParent
-}
-
-task ballerinaPublish {
-
-    ext.balPush = { additionalParams ->
-        exec {
-            workingDir project.projectDir
-            environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
-            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-                commandLine 'cmd', '/c', "$distributionBinPath/bal.bat push ${additionalParams} && exit %%ERRORLEVEL%%"
-            } else {
-                commandLine 'sh', '-c', "$distributionBinPath/bal push ${additionalParams} "
+            copy {
+                from file("$project.projectDir/target/apidocs/${packageName}")
+                into file("$project.projectDir/build/docs_parent/docs/${packageName}")
             }
         }
-    }
 
-    doLast {
-        // Publish to local repository
-        def repository = ""
-        if (project.hasProperty("repository")) {
-            repository = project.findProperty("repository")
-        }
-
-        if (repository == "local") {
-            println("Publishing to the ballerina local repository...")
-            balPush("--repository=local")
-        } else if (repository == "central") {
-            // Publish to central
-            println("Publishing to the ballerina central..")
-            if (project.version.endsWith('-SNAPSHOT') ||
-                    project.version.matches(project.ext.timestampedVersionRegex)) {
+        if (needPublishToCentral) {
+            if (project.version.endsWith('-SNAPSHOT')) {
                 throw new InvalidUserDataException("Cannot push SNAPSHOT version to central")
```

Comment:
Change it to return as well

Reply:
changed in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/132/commits/6a30a525281f84a9aa0f31a7def078a1e0e5141c